### PR TITLE
Add GitHub Issue and Pull request templates

### PR DESCRIPTION
**Pull request by dragetd**
This adds basic templates for issues and pull requests.

Partially implements #243

### CODE REVIEW

**Comment by dragetd** [comment]
There are more advanced template features that allows actual forms (with dropdowns, input-boxes etc.). Do you want to use such a thing or is this enough?

Also, do you want 'other issues'?

I enabled the PR Branch as the default branch on my fork https://github.com/dragetd/headscale, feel free to open test-issues there to test the templates!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!-- Please tick at least one of the following: -->
+
+- [ ] I have read the [CONTRIBUTING guidelines](README.md#user-content-contributing)
+- [ ] A GitHub issue was raised for this feature / bug or it was discussed on the projects chat
```
Comment (kradalby) [inline .github/pull_request_template.md:4]: Could you add the checklist here with some of the items from the issue?

**Review by kradalby — APPROVED** [review]
I think this looks reasonable. Specially to get started :)

File: .github/pull_request_template.md (hunk 1 above)
Comment (dragetd) [reply .github/pull_request_template.md:4]: Oops, sorry. Overlooked that you actually had an example xD

Adding the checkpoints…

**Review by kradalby — APPROVED** [review]

**Comment by kradalby** [comment]
Could you have a look at the formatting issue @dragetd ?

**Comment by dragetd** [comment]
Sorry, not sure what you mean. I was wondering what we were waiting for. :)

Do you mean CI / prettier-lint? Since no code was changed and the Log only reads `ERROR: `, I assumed it was a CI issue unrelated to this PR. Could you explain what you mean, then I am more than happy to update it!

**Comment by kradalby** [comment]
Code and docs must be formatted, so run make fmt, which needs some tools installed. Can be found in makefile

**Comment by dragetd** [comment]
Sorry for overlooking this. Cool tool, I should consider adding it also to some project of mine.

(But calling 'make' gives me PTSD in go projects xD… I am postponing migrating another go project over to `made` instead. :P)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Issues must have some content
+blank_issues_enabled: false
+
+# Contact links
+contact_links:
+  - name: "headsacle usage documentation"
```
Comment (kradalby) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
  - name: "headscale usage documentation"
```

**Review by kradalby — COMMENTED** [review]
I'm happy with this, just one little typo

**Review by kradalby — APPROVED** [review]

**Comment by kradalby** [comment]
I think you did pretty well, I would not have spent this much time on it. Thank you :)

**Pull request merged**